DSESwift/DataPoint.swift

```diff
@@ -55,7 +55,7 @@ public class DataPoint{
      * Set time of the data point.
      * @param time: NSDate for the timestamp of the datapoint.
      **/
-    func setTime(time: NSDate){
+    public func setTimeWithNSDate(time: NSDate){
```

Comment:
Agreed. I hate this as well, but this is inevitable as long as we expose time attribute as read-only property of DataPoint. The conversion from swift to objective-c automatically reserve the function name setTime, so the compiler complains if we use setTime. Sadness.